Gruntfile.js

```diff
@@ -58,12 +58,18 @@ module.exports = function(grunt) {
 							slowTag = '--grep @slow';
 						} else if (tag === 'unstable') {
 							slowTag = '--grep @unstable';
+						} else if (tag === 'network') {
+							slowTag = '--grep @network';
+						} else if (tag === 'propagation') {
+							slowTag = '--grep @propagation';
+						} else if (tag === 'stress') {
+							slowTag = '--grep @stress';
```

Comment:
This variable need to be renamed to reflect its purpose. May be `tagFilter` is appropriate.

Reply:
I will open another issue to review the tagging mechanism.